dcat/index.html

```diff
@@ -4073,6 +4107,9 @@ <h3>Changes since the W3C Recommendation of 16 January 2014</h3>
                 Each relevant class section was extended with properties to deal with licensing, acccess rights, other rights and policies associated with the classes, as
                 recommended in the  <a href="#license-rights">License and rights statements</a> section.
             </li>
+            <li>
```

Comment:
This needs to be removed: it is repeated because I merged the two branches that had similar changes

Reply:
done in 5b88e8a